### PR TITLE
Update CI badges with support for stages

### PR DESCRIPTION
**Pull request by shrijitsingh99**
Updated README with reference to #3795 
Slightly unrelated question, is the PCL logo in the README supposed to be center aligned?
Was wondering cause it had the `align=center` attribute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,19 @@
...
+[ci-macos-10.14]: https://dev.azure.com/PointCloudLibrary/pcl/_apis/build/status/9?branchName=master&stageName=Build%20macOS&jobName=macOS&configuration=macOS%20Mojave%2010.14&label=macOS%20Mojave%2010.14
+[ci-macos-10.15]: https://dev.azure.com/PointCloudLibrary/pcl/_apis/build/status/9?branchName=master&stageName=Build%20macOS&jobName=macOS&configuration=macOS%20Catalina%2010.15&label=macOS%20Catalina%2010.15
+
+Build Platform               | Status                                                                                                                                                                                                                                                                                                                                       
+------------------------ | -------------------------------------------------------------------------------------------- |
+**Ubuntu**               | [![Status][ci-ubuntu-16.04]][ci-latest-build] [![Status][ci-ubuntu-19.10]][ci-latest-build]  |
```
Comment (taketwo) [inline README.md:26]: **Bold** is used in the table header row, I'd leave normal rows without it.

File: README.md [hunk 2]
```diff
@@ -13,15 +13,19 @@
...
-[ci-ubuntu-image]: https://img.shields.io/azure-devops/build/PointCloudLibrary/0fc52e87-00b9-420e-acbc-c842c4f2d9cc/8.svg?label=Ubuntu&logo=azure%20pipelines
-[ci-windows]: https://dev.azure.com/PointCloudLibrary/pcl/_build/latest?definitionId=3
-[ci-windows-image]: https://img.shields.io/azure-devops/build/PointCloudLibrary/0fc52e87-00b9-420e-acbc-c842c4f2d9cc/3.svg?label=Windows&logo=azure%20pipelines
-[ci-macos]: https://dev.azure.com/PointCloudLibrary/pcl/_build/latest?definitionId=2
-[ci-macos-image]: https://img.shields.io/azure-devops/build/PointCloudLibrary/0fc52e87-00b9-420e-acbc-c842c4f2d9cc/2.svg?label=MacOS&logo=azure%20pipelines
+[ci-latest-build]: https://dev.azure.com/PointCloudLibrary/pcl/_build/latest?definitionId=9&branchName=master
+[ci-ubuntu-16.04]: https://dev.azure.com/PointCloudLibrary/pcl/_apis/build/status/9?branchName=master&stageName=Build%20Ubuntu&jobName=Ubuntu&configuration=Ubuntu%2016.04%20GCC&label=Ubuntu%2016.04
+[ci-ubuntu-19.10]: https://dev.azure.com/PointCloudLibrary/pcl/_apis/build/status/9?branchName=master&stageName=Build%20Ubuntu&jobName=Ubuntu&configuration=Ubuntu%2016.04%20GCC&label=Ubuntu%2016.04
```
Comment (taketwo) [inline README.md:18]: Typo in the address here.

**Comment by SergioRAgostinho** [comment]
Since you're touching the badges, you might want to add the link to the discord server.
[Discord](https://discord.gg/gvcqHTW)
```markdown
[![Discord](https://img.shields.io/discord/694824801977630762?color=7289da&label=community%20chat&logo=discord&style=plastic)](https://discord.gg/gvcqHTW)
```

File: README.md [hunk 3]
```diff
@@ -13,15 +13,19 @@
...
+
+Build Platform               | Status                                                                                                                                                                                                                                                                                                                                       
+------------------------ | -------------------------------------------------------------------------------------------- |
+**Ubuntu**               | [![Status][ci-ubuntu-16.04]][ci-latest-build] [![Status][ci-ubuntu-19.10]][ci-latest-build]  |
+**Windows**              | [![Status][ci-windows-x86]][ci-latest-build]  [![Status][ci-windows-x64]][ci-latest-build]   |                                                                                                                                                                    
+**macOS**                | [![Status][ci-macos-10.14]][ci-latest-build]  [![Status][ci-macos-10.15]][ci-latest-build]   |
```
Comment (SunBlack) [inline README.md:28]: Just as suggestion: Add `<br/>` between the badges, so the eye can faster compare both badges from one OS

**Pull request merged**